frontend/src/components/EditMedia.vue

```diff
@@ -40,12 +40,12 @@
       <o-button @click="$emit('close')">Cancel</o-button>
       <div>
         <o-button variant="primary" @click="save">
-          <action-button :mode="loading ? 'loading' : ''">Save</action-button>
+          <action-button :mode="loading ? 'loading' : 'off'">Save</action-button>
         </o-button>
       </div>
     </template>
   </modal>
-  <new-profile
+  <NewProfile
```

Comment:
i've never used vue before so idk why this works when other components can clearly go back-and-forth between camelcase and kebab case, but this does fix it

Reply:
interesting, i've never run into that before